airflow/ui/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Heading } from "@chakra-ui/react";
+
+import { Health } from "./Health";
+
+export const Dashboard = () => (
+  <Box>
+    <Heading mb={4}>Welcome</Heading>
+
+    <Box>
+      <Health />
```

Comment:
Minimal bootstrap. Good start. Looking forward having a configurable dashboard with modules an admin (or even a user) can define in future :-D

Reply:
Super minimal, but we should set up a responsive Grid layout once we have multiple components

Reply:
So many ideas jumping in my head immediately... but getting basics first!